WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BlockListItem.kt

```diff
@@ -85,6 +85,7 @@ sealed class BlockListItem(val type: Type) {
         val subText: String? = null,
         @StringRes val valueResource: Int? = null,
         val value: String? = null,
+        val percentageOfMaxValue: Double? = null,
```

Comment:
I'd rename this to `barWidth` because the view doesn't really care what the bar represents. I also think it should be an integer.

Reply:
Makes sense.  Will do.